src/Lime.Messaging/Contents/Reaction.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Lime.Protocol;
+using System.Runtime.Serialization;
+using static System.Net.WebRequestMethods;
+
+namespace Lime.Messaging.Contents
+{
+    [DataContract(Namespace = "http://limeprotocol.org/2014")]
+    public class Reaction : Document
+    {
+        public const string MIME_TYPE = "application/vnd.lime.reaction+json";
+        public static readonly MediaType MediaType = MediaType.Parse(MIME_TYPE);
+
+        public const string EMOJIS_KEY = "emojis";
```

Comment:
EMOJIS_KEY is used to set the name of a Data Member(Emoji). Maybe this constant should be in the singular.